### PR TITLE
Add basic description of Device Farm

### PR DESCRIPTION
**Pull request by artemnikitin**

### CODE REVIEW

**Comment by eventi** [comment]
This is a great start. LGTM

File: README.md [hunk 1]
```diff
@@ -1254,6 +1255,17 @@ EMR
 -	Python programmers may want to take a look at Yelp’s [mrjob](https://github.com/Yelp/mrjob).
 -	It takes time to tune performance of EMR jobs, which is why third-party services such as [Qubole’s data service](https://www.qubole.com/mapreduce-as-a-service/) are gaining popularity as ways to improve performance or reduce costs.
 
+DeviceFarm
```
Comment (jlevy) [inline README.md]: Space

**Comment by jlevy** [comment]
And thanks for the additions!

File: README.md [hunk 2]
```diff
@@ -1254,6 +1255,17 @@ EMR
...
+----------
+
+### Device Farm Basics
+-	📒 [Homepage](https://aws.amazon.com/device-farm/) ∙ [Developer guide](http://docs.aws.amazon.com/devicefarm/latest/developerguide/) ∙ [FAQ](https://aws.amazon.com/device-farm/faq/) ∙ [Pricing](https://aws.amazon.com/device-farm/pricing/)
+- **Device Farm** is AWS service for mobile app testing on real devices.
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: "..is an AWS service that enables mobile app testing..."

File: README.md [hunk 3]
```diff
@@ -1254,6 +1255,17 @@ EMR
...
+- Support devices on iOS and Android, including Kindle Fire and mobile web.
+- Another feature is remote access to a particular device for interactive testing\debugging session.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "Supports remote device access in order to allow for interactive testing/debugging."

File: README.md [hunk 4]
```diff
@@ -1254,6 +1255,17 @@ EMR
 -	Python programmers may want to take a look at Yelp’s [mrjob](https://github.com/Yelp/mrjob).
 -	It takes time to tune performance of EMR jobs, which is why third-party services such as [Qubole’s data service](https://www.qubole.com/mapreduce-as-a-service/) are gaining popularity as ways to improve performance or reduce costs.
 
+DeviceFarm
+----------
+
+### Device Farm Basics
+-	📒 [Homepage](https://aws.amazon.com/device-farm/) ∙ [Developer guide](http://docs.aws.amazon.com/devicefarm/latest/developerguide/) ∙ [FAQ](https://aws.amazon.com/device-farm/faq/) ∙ [Pricing](https://aws.amazon.com/device-farm/pricing/)
+- **Device Farm** is AWS service for mobile app testing on real devices.
+- Support devices on iOS and Android, including Kindle Fire and mobile web.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "Supports iOS and Android (including Kindle Fire) devices, as well as the mobile web."

File: README.md (hunk 1 above)
Comment (artemnikitin) [inline README.md]: Do you mean space between previous service description and Device Farm or space on the line with "DeviceFarm" word?

**Comment by artemnikitin** [comment]
Apply suggestions from @ThanosBaskous

**Comment by artemnikitin** [comment]
> do you think you can expand this section further?
Yes, I'm planning to add more info in coming days.

File: README.md [hunk 5]
```diff
@@ -1254,6 +1255,17 @@ EMR
 -	Python programmers may want to take a look at Yelp’s [mrjob](https://github.com/Yelp/mrjob).
 -	It takes time to tune performance of EMR jobs, which is why third-party services such as [Qubole’s data service](https://www.qubole.com/mapreduce-as-a-service/) are gaining popularity as ways to improve performance or reduce costs.
 
+DeviceFarm
+----------
+
+### Device Farm Basics
+-	📒 [Homepage](https://aws.amazon.com/device-farm/) ∙ [Developer guide](http://docs.aws.amazon.com/devicefarm/latest/developerguide/) ∙ [FAQ](https://aws.amazon.com/device-farm/faq/) ∙ [Pricing](https://aws.amazon.com/device-farm/pricing/)
+- **Device Farm** is an AWS service that enables mobile app testing on real devices.
+- Supports iOS and Android (including Kindle Fire) devices, as well as the mobile web."
```
Comment (artemnikitin) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (artemnikitin) [inline README.md]: Fixed

**Comment by artemnikitin** [comment]
Fixed all issues

**Comment by mahadeokr** [comment]
IMEI353310076723556

**Comment by artemnikitin** [comment]
@mahadeokr what do you mean ?

**Comment by artemnikitin** [comment]
It seems I broke this pull request while resolving conflicts. I will create a new one.

**Pull request closed without merging**